assets/js/components/dashboard-sharing/DashboardSharingSettings/Footer.js

```diff
@@ -79,25 +85,45 @@ export default function Footer( { closeDialog } ) {
 		closeDialog();
 	}, [ closeDialog, viewContext ] );
 
+	const showNotice =
+		errorNotice ||
+		haveSharingSettingsChangedManagement ||
+		haveSharingSettingsChangedRoles;
+
 	return (
 		<div className="googlesitekit-dashboard-sharing-settings__footer">
-			<div className="googlesitekit-dashboard-sharing-settings__footer-notice">
-				{ errorNotice && <ErrorText message={ errorNotice } /> }
-				{ ! errorNotice && <Notice /> }
-			</div>
+			{ showNotice && (
+				<div className="googlesitekit-dashboard-sharing-settings__footer-notice">
+					{ errorNotice && <ErrorText message={ errorNotice } /> }
+					{ ! errorNotice && <Notice /> }
+				</div>
+			) }
 
 			<div className="googlesitekit-dashboard-sharing-settings__footer-actions">
-				<Link onClick={ onCancel }>
-					{ __( 'Cancel', 'google-site-kit' ) }
-				</Link>
-
-				<Button
-					onClick={ onApply }
-					disabled={ isSaving || ! canSubmitSharingChanges }
-				>
-					{ __( 'Apply', 'google-site-kit' ) }
-					{ isSaving && <Spinner isSaving={ isSaving } /> }
-				</Button>
+				<div className="googlesitekit-dashboard-sharing-settings__footer-actions-left">
+					<Link
+						onClick={ () => {
+							openResetDialog();
+						} }
```

Comment:
This is probably fine here:

```suggestion
						onClick={ openResetDialog }
```